components/conversations/Comment.js

```diff
@@ -42,6 +42,7 @@ const Comment = ({
   onDelete,
   reactions,
   canReply,
+  getClickedComment,
```

Comment:
The other prop name that you've used looks clearer IMO
```suggestion
  onReplyClick,
```